### PR TITLE
Remove tutorials

### LINKED ISSUE #122: Simple multi-layer perceptron tutorial does not work on macOS Monterey
Per https://github.com/JuliaML/MLDatasets.jl/issues/86 this tutorial no longer works with macOS Monterey

### LINKED ISSUE #132: getting started webpage error
in the [getting started page](https://fluxml.ai/getting_started.html) the command `ps = params(W, b)` returns
```
ERROR: UndefVarError: params not defined
Stacktrace:
 [1] top-level scope
   @ REPL[17]:1
 [2] top-level scope
   @ ~/.julia/packages/CUDA/GGwVa/src/initialization.jl:52
```
while `ps = Flux.params(W, b)` works correctly.
maybe fix the getting started page.

### PR DESCRIPTION
**Pull request by mcabbott**
After https://github.com/FluxML/Flux.jl/pull/2125 I believe we can delete the tutorial section here completely. Then we have code in two places, the docs + the model zoo, not three. As desired for instance here: https://github.com/FluxML/fluxml.github.io/issues/141#issuecomment-1268466522 

Besides the 5 moved to docs already:

* Maybe https://fluxml.ai/tutorialposts/2020-10-18-transfer-learning/ should also be copied over & updated? Seems a bit thin, as noted in #123 
* We can delete https://fluxml.ai/tutorialposts/2021-01-21-data-loader/ as it's basically just the docstring... or add a preamble to https://fluxml.ai/Flux.jl/stable/data/mlutils/ 
* Not in tutorials, but  https://fluxml.ai/getting_started/ is also content we have so many other places, such as https://fluxml.ai/Flux.jl/stable/models/overview/ .

This PR also tries to clean up the navbar. The code had a baroque set of switches apparently to deal with relative paths from various places. I'm not certain what it should contain, but the first attempt is this (with current state on top, PR below):

Screenshot 2022-11-26 at 23 33 16

At present that has no links to docs sub-sections. But perhaps it ought not to link to stack overflow / discourse, those are just links not things Flux owns. And could then instead have Ecosystem?

Edit, I see that https://github.com/FluxML/fluxml.github.io/pull/140 is also quite recent WIP. But this aims for the blog here, not the tutorials, I think.

Closes #122
Closes #132

### CODE REVIEW

**Comment by mcabbott** [comment]
Attempting to add redirects in last commit. But  https://fluxml.ai/previews/PR156/getting_started/ doesn't work, although the file looks like https://github.com/FluxML/fluxml.github.io/blob/main/blogposts/2017-08-24-generic-gpu.md which does redirect. Is this a preview limitation?

**Comment by mcabbott** [comment]
Since Flux 0.13.9 is tagged, this ought to be safe to merge.

However, it would be nice if the redirection worked. 

I'm also not sure they point to the right place. Maybe some should point to the model zoo instead.

**Comment by Saransh-cpp** [comment]
> The code had a baroque set of switches apparently to deal with relative paths from various places.

This code is a bit annoying, but unfortunately, this is what keeps the previews working 🙂

Given that we store previews within the gh-pages branch (to avoid the extra Netlify infrastructure), it becomes important to ensure that the previews don't access the content of the stable website and vice-versa.

**Comment by Saransh-cpp** [comment]
For instance, if you go to the [preview](https://fluxml.ai/previews/PR156/) and click the "blog" button on the navbar, it will redirect you to the "blog" section of the stable website (because this PR now does not use relative links).

**Comment by mcabbott** [comment]
I see. Is there any way to do that without duplicating the content? Define some variable which is either `""` or `"../"` at the top of the page & splice that in?

Otherwise having to typing the right path into the online preview (but not local ones) might still be preferable to having multiple copies of the same content which can have different mistakes...

**Comment by Saransh-cpp** [comment]
> Is there any way to do that without duplicating the content?

I'll try that! Stuck with my end semester examinations at the moment; we could either switch back to the weird `if-else` blocks, or keep it like this (as you suggest) for this PR.

**Comment by Saransh-cpp** [comment]
> Attempting to add redirects in last commit. But https://fluxml.ai/previews/PR156/getting_started/ doesn't work, although the file looks like https://github.com/FluxML/fluxml.github.io/blob/main/blogposts/2017-08-24-generic-gpu.md which does redirect. Is this a preview limitation?

Not very sure what is happening here. It should've worked. Will look into this.

**Comment by mcabbott** [comment]
If restored, the if-else block need only wrap a few entries like blog, rather than duplicating everything. 

You don't have any ideas why the redirects are failing, by the way?

**Comment by Saransh-cpp** [comment]
As far as I can see, the GH Actions are building the redirected endpoints - https://github.com/FluxML/fluxml.github.io/tree/gh-pages/previews/PR156.

But, no idea why deployment is not showing them.

**Comment by Saransh-cpp** [comment]
Oh wait, I think the redirects do not happen automatically, rather - https://github.com/FluxML/fluxml.github.io/blob/main/utils.jl - is used to make everything work.

**Comment by Saransh-cpp** [comment]
Ah! This PR tries to redirect some pages, but there is no default redirect (or `external`) command. I went through Franklin's documentation and found nothing on it.

The GH Actions are producing the HTML files, but these HTML files are empty (as the markdown files are empty; Franklin does not understand `external`).

The website, right now, uses `utils.jl` to provide redirects for blogs and tutorials (`utils.jl` understands `external`). We should write a similar script for these lost pages, or we could copy paste the pages from Flux's documentation and the endpoints will work.

**Comment by mcabbott** [comment]
Are you saying that `hfun_recentposts` in utils.jl is run only on the blog posts (whose redirects via external seem to work) and not on the tutorialposts?

**Comment by Saransh-cpp** [comment]
I might have explained the existing mechanism wrong. Technically, no "redirection" is involved in the blog posts and tutorials. The function `hfun_recentposts` is responsible for writing the https://fluxml.ai/blog/ and https://fluxml.ai/tutorials/ pages. The links on these pages are given the `external` variable's value.

For instance, if you go to https://fluxml.ai/blogposts/2018-12-03-ml-language-compiler/ (blog hosted somewhere else), the website will not redirect you. You will be shown an empty web page because the markdown file is empty. But, if you click on "Building a Language and Compiler for Machine Learning (December 2018)" on the "blogs" page, you will land on the correct URL (not redirection, the link itself is the link where the blog is hosted).

**Comment by mcabbott** [comment]
Oh right, sorry, I see now. 

Ok, so copying that isn't really the problem. We don't need a page here which links to the new locations. 

The only reason to want redirects is in case people elsewhere have links to the existing pages. (Or google might.) Such redirects would have to be made by a different mechanism.

**Comment by Saransh-cpp** [comment]
I'll check on Franklin's GitHub or Slack if there is a nice way to do this.

File: _layout/navbar.html [hunk 1]
```diff
@@ -1,117 +1,39 @@
...
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" href="https://github.com/FluxML/model-zoo/" target="_blank">Model Zoo</a>
       </li>
       <li class="nav-item">
         <a class="nav-link" href="https://github.com/FluxML/Flux.jl" target="_blank">GitHub</a>
       </li>
       <li class="nav-item">
-        <a class="nav-link" href="https://stackoverflow.com/questions/tagged/flux.jl" target="_blank">Stack Overflow</a>
+        <a class="nav-link" href="https://fluxml.ai/Flux.jl/dev/ecosystem/">Ecosystem</a>
```
Comment (mcabbott) [inline _layout/navbar.html:22]:
```suggestion
        <a class="nav-link" href="https://fluxml.ai/Flux.jl/stable/ecosystem/">Ecosystem</a>
```

**Comment by mcabbott** [comment]
Bump -- https://twitter.com/owainkenway/status/1619003403765694465 is a recent complaint which I'm told is about old Flux tutorials being misleading.

**Comment by Saransh-cpp** [comment]
My message seems to have gone up on slack. I hope tagging @tlienart here would be okay.

Just to reiterate, we want to redirect users from some specific endpoint to another. For instance, https://clusterinnovationcentre.github.io will automatically redirect you to https://clusterinnovationcentre.github.io/convoke/2023. This happens because `index.html` has the following contents -
```html
<meta http-equiv="refresh" content="0; url=./2023/"/>
```

Can we perform such a redirect (another website redirect instead of a relative URL redirect) using `Franklin`? Maybe something like a "redirect" variable?

**Comment by Saransh-cpp** [comment]
Thank you! That should do!

**Comment by mcabbott** [comment]
[149a8bb](https://github.com/FluxML/fluxml.github.io/pull/156/commits/149a8bbe2b358bd32fa6795fe04bb70d9e766c35) adds something like that. But going to  https://fluxml.ai/previews/PR156/tutorialposts/2021-10-14-vanilla-gan/  still loads a stub page here, rather than a page in the docs.

**Comment by mcabbott** [comment]
Thanks for taking a look. 

But is this the right way around? The linked docs say "when a user goes to `(baseurl)/some/other/addr.html`, they will be redirected to the current page" which sound like this code should be on the new page (where you want readers to land). 

The meta tag [above](https://github.com/FluxML/fluxml.github.io/pull/156#issuecomment-1409020312) is instead placed on the old page (where you're removing content) to send readers elsewhere.

The pages we would ideally like to send readers to are either in Flux's docs (generated by Documenter) or else in the [model-zoo](https://github.com/FluxML/model-zoo/blob/master/tutorials/60-minute-blitz/60-minute-blitz.jl), a separate repository with README pages.

**Comment by mcabbott** [comment]
Ok I think I understand. But the point of this PR is no longer to attempt to host this code within the website at all (since it goes stale here). 

The hope was to send anyone following a link here along to either the model zoo (different domain, github README) or else to a page in the docs (same domain, Documenter). I believe that's what Saransh's meta code would do -- there isn't some way to embed a literal tag?

Or can we use `.htaccess` here?

**Review by darsnack — APPROVED** [review]
I think this is a good idea. I would also remove "Discourse / Stack Overflow" for a link to "Ecosystem." The latter should be front and center IMO since we get a lot of people making feature requests or posts elsewhere for things that are in existing (not Flux.jl) packages.

**Comment by mcabbott** [comment]
Great, the present state of this PR has these headings:

Screenshot 2023-02-08 at 14 12 30

I am giving up on making redirects work; someone else can try (in another PR) if strongly desired. The main thing is not to leave bad & outdated examples up in such an official place.

**Comment by mcabbott** [comment]
Ah in fact  https://fluxml.ai/previews/PR156/tutorialposts/2021-10-14-vanilla-gan/  does now redirect. I don't know what changed. But [1d13aa4](https://github.com/FluxML/fluxml.github.io/pull/156/commits/1d13aa4ff233e435403799b78083a10c1eaa2ce4) could then be done for other pages too. In another PR.

**Comment by darsnack** [comment]
Weird, I didn't see it in the preview earlier today, but I see it now.

**Comment by mcabbott** [comment]
Weird indeed.

Thanks @tlienart for your help with this!

**Pull request merged**